src/pages/about-us/style.less

```diff
@@ -33,6 +33,40 @@
   }
 }
 
+.board-members {
+  label {
+    display: inline-flex;
+    padding: 1em 0;
+    font-size: 3em;
+    font-weight: bolder;
+  }
+
+  select {
```

Comment:
When you select an option theres a blue outline on chrome you can solve it using [this quick css fix](https://stackoverflow.com/questions/31570444/remove-blue-highlighting-of-option)
![image](https://user-images.githubusercontent.com/8893880/80935517-d4e07980-8d81-11ea-88b2-cd9b5890192a.png)